src/python/pants/engine/internals/scheduler.py

```diff
@@ -358,6 +358,21 @@ def lease_files_in_graph(self, session):
     def garbage_collect_store(self):
         self._native.lib.garbage_collect_store(self._scheduler)
 
+    def nailgun_server_await_bound(self, nailgun_server) -> int:
+        """Blocks until the server has bound a port, and then returns the port.
+
+        Returns the actual port the server has successfully bound to, or raises an exception if the
+        server has exited.
+        """
+        return cast(int, self._native.nailgun_server_await_bound(self._scheduler, nailgun_server))
+
+    def new_nailgun_server(self, port_requested: int, runner: RawFdRunner) -> Any:
```

Comment:
Is this `Any` because it truly could be `Any`, or because you don't know the more precise type? If the latter, drop the return type.

Reply:
We don't know the precise type because it is coming out of CFFI.